src/rules/onlyArrowFunctionsRule.ts

```diff
@@ -55,7 +55,7 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class OnlyArrowFunctionsWalker extends Lint.RuleWalker {
     public visitFunctionDeclaration(node: ts.FunctionDeclaration) {
-        if (this.getOptions().indexOf(OPTION_ALLOW_DECLARATIONS) === -1) {
+        if (this.hasOption(OPTION_ALLOW_DECLARATIONS)) {
```

Comment:
my bad, I should have suggested `!this.hasOption`, I think the logic got inverted here